.github/workflows/ci.yml

```diff
@@ -3,6 +3,7 @@ name: Keycloak CI
 on: [push, pull_request]
 
 env:
+  DEFAULT_JDK_DISTRIBUTION: temurin
```

Comment:
For more information about this option see [the documentation](https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#Selecting-a-Java-distribution).